src/cco-modules/EventOntology.ttl

```diff
@@ -2116,7 +2116,9 @@ cco:ont00000902 rdf:type owl:Class ;
 cco:ont00000908 rdf:type owl:Class ;
                  rdfs:subClassOf cco:ont00000228 ;
                  rdfs:label "Act of Artifact Processing"@en ;
-                 skos:definition "A Planned Act of performing a series of mechanical or chemical operations on something in order to change or preserve it."@en ;
+                 skos:definition "A Planned Act of performing a series of physical operations on an Artifact."@en ;
+                 cco:ont00001754 "Wikipedia contributors, 'Industrial processes,'' Wikipedia, The Free Encyclopedia, https://en.wikipedia.org/w/index.php?title=Industrial_processes&oldid=1246012106 (accessed April 29, 2025)."^^xsd:anyURI ;
+                 skos:scopeNote "'Physical' should be understood in a broad manner that especially encompasses mechanical and chemical operations. A mechanical operation may be the polishing of a piece of metal, and a chemical operation may be the anodizing of a piece of metal."@en ;
```

Comment:
Suggest moving the examples out of scopeNote to use the example annotation instead.